components/application/index.jsx

```diff
@@ -11,9 +11,11 @@ const Application = ({ children, dataProvider, activity, ...restProps }) => (
   <>
     <Head />
     <Container {...restProps}>
-      <AppBar>
-        <RepositorySelect value={dataProvider} />
-      </AppBar>
+      {dataProvider && (
+        <AppBar>
+          <RepositorySelect value={dataProvider} />
+        </AppBar>
+      )}
```

Comment:
You hide whole application bar if data provider is not set. It would be better to hide only RepositorySelect. In application bar can be other menu, user area etc.